build.gradle

```diff
@@ -5,6 +5,10 @@ repositories {
 }
 
 dependencies {
+    compile 'com.fasterxml.jackson.core:jackson-core:2.6.3'
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.6.3'
+    compile 'com.fasterxml.jackson.core:jackson-annotations:2.6.3'
```

Comment:
I believe we don't need `jackson-annotations` here. I'm also unsure about `jackson-databind`, but less certain.